pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -964,6 +967,109 @@ type MSTeamsConfig struct {
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
 }
 
+// RocketChatConfig configures notifications via RocketChat.
+// See https://prometheus.io/docs/alerting/latest/configuration/#rocketchat_config
+type RocketChatConfig struct {
+	// Whether to notify about resolved alerts.
+	// +optional
+	SendResolved *bool `json:"sendResolved,omitempty" yaml:"send_resolved,omitempty"`
+	// The API URL for RocketChat.
+	// Defaults to global.rocketchat_api_url if not specified.
+	// +optional
+	APIURL *string `json:"apiURL,omitempty" yaml:"api_url,omitempty"`
+	// The channel to send alerts to.
+	// Defaults to global.rocketchat_channel if not specified.
+	// +optional
+	Channel *string `json:"channel,omitempty" yaml:"channel,omitempty"`
+	// The sender token (mutually exclusive with TokenFile).
+	// +optional
+	Token v1.SecretKeySelector `json:"token,omitempty" yaml:"token,omitempty"`
+	// The file containing the sender token (mutually exclusive with Token).
```

Comment:
Also would be more clear if instead of mutually exclusive you type as `token` or `tokenFile` is required, same applies for `tokenid` and `tokenid` file